crates/gs_common/src/network/server.rs

```diff
@@ -632,4 +642,68 @@ impl rpc::authenticated_server_connection::Server for RcAuthenticatedServer2Clie
         );
         Promise::ok(())
     }
+
+    fn send_throw_action(&mut self, params: SendThrowActionParams, _: SendThrowActionResults) -> Promise<(), Error> {
+        let params = pry!(params.get());
+        let position: PositionData = pry!(params.get_position()).try_into().unwrap();
```

Comment:
Ideally I want to avoid copying data from capnp structs to intermediate rust structs that just repackage the data, a big benefit of capnp is avoiding memory copies and doing this here introduces a needless memory copy. For a couple of Vec3s this doesn't matter much, but it'd be nice to be consistent with this